app/config/services.yml

```diff
@@ -5,6 +5,12 @@ services:
     cache.null_provider:
         class: Symfony\Component\Cache\Adapter\NullAdapter
 
+    cache.bbc:
+        class: BBC\ProgrammesPagesService\Cache\Cache
+        arguments:
+            - '@cache.programmes'
+            - ''
```

Comment:
We'll want the app name, and release number

 `- 'clifton.%cosmos_component_release%'`